website/snippets/_packages_or_dependencies.md

```diff
@@ -1,16 +1,23 @@
 
 ## Use cases
 
-Starting from dbt v1.6, we added a new configuration file called `dependencies.yml`. The file can contain both types of dependencies: "package" and "project" dependencies.
-- ["Package" dependencies](/docs/build/packages#how-do-i-add-a-package-to-my-project) lets you add source code from someone else's dbt project into your own, like a library.
-- ["Project" dependencies](/docs/collaborate/govern/project-dependencies) provide a different way to build on top of someone else's work in dbt.
+The following setup will work for every dbt project:
+
+- Add [any package dependencies](/docs/collaborate/govern/project-dependencies#when-to-use-project-dependencies) to `packages.yml`
+- Add [any project dependencies](/docs/collaborate/govern/project-dependencies#when-to-use-package-dependencies) to `dependencies.yml`
+
+However, you can consolidate both into a single `dependencies.yml` file. Read the following section to learn more.
```

Comment:
The current wording makes it seem like it **will** be possible to consolidate into a single `dependencies.yml`, so I'm wondering if we can soften the language here by using **may** (or **might**) instead?
```suggestion
However, you may be able to consolidate both into a single `dependencies.yml` file. Read the following section to learn more.
```